include/envoy/server/admin.h

```diff
@@ -15,6 +16,39 @@
 namespace Envoy {
 namespace Server {
 
+class AdminStream {
+public:
+  virtual ~AdminStream() {}
+
+  /**
+   * @param end_stream set to false for streaming response. Default is true, which will close
+   * connection on socket.
+   */
+  virtual void setEndStreamOnComplete(bool end_stream) PURE;
+
+  /**
+   * @param cb callback to be added to the list of callbacks invoked by onDestroy() when connection
+   * is dropped.
```

Comment:
Same comment. Connection is not necessarily dropped. When the stream is closed is more accurate.